commands/bmc.vegman

```diff
@@ -117,10 +117,11 @@ function cmd_datetime_method {
   method=${method%\"*}
 
   if [[ $# -eq 0 ]]; then
-      echo "Current time synchronization method is ${method}"
+      # VEGMAN doesn't have RTC and in manual mode it uses PCH time.
+      echo "Current time synchronization method is ${method/Manual/HostTime}"
```

Comment:
So now we have it like this?

```
$ bmc datetime method host
$ bmc datetime method
Current time synchronization method is HostTime
```

Doesn't this looks strange to you?
Let's make it `Host` here so that it matches the command argument.

Reply:
done